.github/ISSUE_TEMPLATE/bug.yml

```diff
@@ -8,16 +8,16 @@ body:
         Thanks for taking the time to fill out this bug report!
 
         > [!WARNING]
-        > This is the new native Bitwarden Beta app repository. For the pubicly available apps in App Store / Play Store, submit your report in [bitwarden/mobile](https://github.com/bitwarden/mobile)
+        > This is the new native Bitwarden app repository. For the pubicly available apps in App Store / Play Store, submit your report in [bitwarden/mobile](https://github.com/bitwarden/mobile)
```

Comment:
🤔  I think we should also remove the report submission, given that the publicly available apps on iOS is this one, the native app.

Reply:
Ah, good call! I didn't read it as closely as I should have!